cmd/osv-scanner/main_test.go

```diff
@@ -576,12 +576,12 @@ func TestRun_Licenses(t *testing.T) {
 		{
 			name: "No vulnerabilities with license summary",
 			args: []string{"", "--experimental-licenses-summary", "./fixtures/locks-many"},
-			exit: 0,
+			exit: 1,
```

Comment:
Should we just bump the version in the relevant lock files in this dir instead? otherwise we're no longer testing the same thing.

Reply:
`busybox` version `1.35.0-r29` has a vulnerability in the test snapshot which I am not sure if we want to keep this version for some testing purpose. 

also bumping the version `alpine.cdx.xml` includes changing `cpe`, `purl` and `properties`, so I mentioned @another-rex to see if he could help with preparing the new fixture.